pages/linux/lsblk.md

```diff
@@ -0,0 +1,27 @@
+# lsblk
+
+> lsblk lists information about all available or the specified storage devices.
+
+- General syntax
+
+`lsblk [options] {{[ device... ]}}`
+
+- Also list empty devices
+
+`lsblk −a`
+
+- Print the SIZE column in bytes rather than in a human-readable format.
+
+`lsblk −b`
+
+- Output info about filesystems.
+
+`lsblk −f`
+
+- Use ASCII characters for tree formatting.
+
+`lsblk −i`
+
+- Produce output in the form of a list.
```

Comment:
So just out of curiosity: this option gives me exactly the same as without. Is the list form the default one?